src/Form/Mixin/Converter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Kirby\Form\Mixin;
+
+use Kirby\Exception\InvalidArgumentException;
+use Kirby\Toolkit\Str;
+
+trait Converter
+{
+	/**
+	 * The converter to apply to the field value
+	 */
+	protected string|null $converter;
+
+	public function converter(): string|null
+	{
+		return $this->converter;
+	}
+
+	public function converters(): array
+	{
+		return [
+			'lower'   => function ($value) {
+				return Str::lower($value);
+			},
+			'slug'    => function ($value) {
+				return Str::slug($value);
+			},
+			'ucfirst' => function ($value) {
+				return Str::ucfirst($value);
+			},
+			'upper'   => function ($value) {
+				return Str::upper($value);
+			},
+		];
+	}
+
+	public function convert(mixed $value): mixed
```

Comment:
I would list all allowed types here. 

```suggestion
	public function convert(string|array|null $value): mixed
```